fl4health/clients/basic_client.py

```diff
@@ -99,6 +99,7 @@ def __init__(
         self.test_loader: Optional[DataLoader]
         self.num_train_samples: int
         self.num_val_samples: int
+        self.num_test_samples: Optional[int]
```

Comment:
I'd set this to None by default just to guarantee an expected value.

Reply:
Ok!